lib/net/peer.js

```diff
@@ -205,13 +214,27 @@ class Peer extends EventEmitter {
 
   /**
    * Getter to retrieve fullname.
+   * Includes identitykey@ip:port.
    * @returns {String}
    */
 
   fullname() {
     return this.address.fullname();
   }
 
+  /**
+   * Getter to retrieve realname. Returns
+   * the fullname when encrypted, otherwise
+   * the hostname.
+   * @returns {String}
+   */
+
```

Comment:
We could opt to just using `fullname` instead of adding `realname`, `fullname` will never render the key. `realname` is more useful if we ever want to choose to create an outbound connection in plaintext when we also have the key.

Reply:
oh yeah, you've got this already - so maybe we dont need to add identity key again in `getpeerinfo`

Reply:
this already looks ok:

```   
{
    addr: 'aorsxa4ylaacshipyjkfbvzfkh3jhh4yowtoqdt64nzemqtiw2whk@127.0.0.1:51363',
    addrlocal: 'aorsxa4ylaacshipyjkfbvzfkh3jhh4yowtoqdt64nzemqtiw2whk@127.0.0.1:10006',
    encrypted: true,
    ...
  },
  {
    id: 2,
    addr: '127.0.0.1:51368',
    addrlocal: '127.0.0.1:10005',
    encrypted: false,
    ...
```